ste/xfer-deleteBlob.go

```diff
@@ -43,12 +43,14 @@ func DeleteBlobPrologue(jptm IJobPartTransferMgr, p pipeline.Pipeline, pacer pac
 		jptm.ReportTransferDone()
 	}
 
-	_, err := srcBlobURL.Delete(jptm.Context(), azblob.DeleteSnapshotsOptionNone, azblob.BlobAccessConditions{})
+	// TODO confirm whether we should add a new flag to allow user to specify whether snapshots should be removed
```

Comment:
Hey wait, didn't we have a hidden flag (--with-snapshots) that never worked?

I removed that in another (un-merged) PR, but I could undo the removal of it.

Reply:
That's fine for now, I'm trying to clarify with them whether they want the snapshot options.

Reply:
I'll pop a small note into the changelog about this too. You'll see it when you review that PR. (Which I've just added you to as a reviewer).